examples/ethereum-tracker/src/lib.rs

```diff
@@ -40,6 +41,17 @@ pub struct InstantiationArgument {
     pub ethereum_endpoint: String,
     /// The address of the contract
     pub contract_address: String,
+    /// The block height at which the EVM contract was created
+    pub start_block: u64,
+}
+
+/// The initialization data required to create an Ethereum tracker
```

Comment:
…without the start block.
But why have that type at all? Why not just make the endpoint and the address two separate fields in the state?

Reply:
We definitely can do that. I am of the opinion, that we should decrease the number of `RegisterView` entries, but that is pretty neutral.